en/orm/saving-data.rst

```diff
@@ -759,6 +759,8 @@ table class. Behaviors and other event subscribers can use the
 ``Model.buildRules`` event to augment the rules checker for a given Table
 class::
 
+    use Cake\ORM\RulesChecker
```

Comment:
no, you can adda commit to this branch and it will automatically update the pull request.
If youre using the online editor, you can go on github, select the branch-32 and edit again the file then commit your modification.
Then if you are motivated to contribute frequently, I would suggest to install git localy. It's pretty straightforward. Basics are described [here in the book](http://book.cakephp.org/3.0/en/contributing/code.html) and I will be happy to help you if you have any question through irc tomorrow during the day for example